qdrant_client/async_qdrant_remote.py

```diff
@@ -1164,7 +1165,7 @@ async def scroll(
                     read_consistency=consistency,
                     shard_key_selector=shard_key_selector,
                 ),
-                timeout=self._timeout,
```

Comment:
this timeout should be `timeout or self._timeout if timeout is None` (or similar to this, an example can be found in methods like `search`)